sda-commons-shared-otel-agent/README.md

```diff
@@ -0,0 +1,6 @@
+# SDA Commons Shared OpenTelemetry agent
+
+This provides an extension of the [OpenTelemetry Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation) that redefines the default configuration that are set in the base agent to match requirements of Sda services.
+All configuration that can be used can be found in [OpenTelemetry SDK Autoconfigure](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure). 
+
+> **_NOTE:_** This module _must_ be built as a fat jar.
```

Comment:
What does that mean for users?

Reply:
You are right. that is only relevant to devs. I will add notes for users.

Reply:
added a reference to the other module, which describes steps to use.